packages/shopping/src/__tests__/acceptance/authorization.acceptance.ts

```diff
@@ -38,9 +31,14 @@ describe('authorization', () => {
     if (app != null) await app.stop();
   });
 
-  describe('customer_service', () => {
-    it('allows customer_service create orders', async () => {
-      newUser = await createAUser();
+  describe('Customer Support', () => {
+    it('does not allows customer support to create orders', async () => {
+      newUser = await createAUser({
+        email: 'support@loopback.io',
```

Comment:
In earlier commits we removed `email`. So we're putting it back in?

Reply:
Why is it in mocha tests but not web ui or server code?

Reply:
Email is required for creating the user. The place where I removed is UserProfile which is used from creating the access token, we have no need for email there.

Reply:
When up until now, the unique identifier in shopping was a unique email. But a unique userid is fine too. when the rbac csv file was there, bob and alice weren't sufficient for my taste. But now that we have unique user id , then this is fine. we don't need email in the token. thx. @hacksparrow .